.github/workflows/compile.yml

```diff
@@ -13,16 +13,22 @@ jobs:
         uses: actions/checkout@v3
         
       - name: Install Dependencies
-        run: sudo apt install -y p7zip-full binutils-mips-linux-gnu build-essential pkgconf python3 git zip
+        run: sudo apt install -y p7zip-full binutils-mips-linux-gnu build-essential pkgconf python3 git zip jq wget curl
 
       - name: Setup Emscripten
         run: "git clone https://github.com/emscripten-core/emsdk.git .emsdk && cd .emsdk && ./emsdk install latest && ./emsdk activate latest && source ./emsdk_env.sh && wget https://raw.githubusercontent.com/EmulatorJS/build/main/emscripten.patch && patch -u -p0 -i emscripten.patch && cd ../"
```

Comment:
We should probably install via my debian repo - since Ive had a broken emscripten commit break this script entirely

Reply:
Agreed... I found an error applying the em patch when I tried building locally yesterday.

Reply:
Resolved this error by pinning the .emsdk version to 3.1.54. That's the last version that the patch works for.